certified-connectors/DocTOPDFConnector/Readme.md

```diff
@@ -0,0 +1,13 @@
+This custom connector is used to convert MS Word document to PDF file with fidelity in imbedded charts, graphs and graphics. This connector calls our company API service which passes the document and credentials , the API service receives the MS Word document and all content and convert it PDF file., once the conversion is completed the API send it back as a PDF to the connector / environment, depending on the configuration of the workflow the file is then deposited in the identified location. We created this tool for our clients to serve a need and feel this may be a connector that has a space in the market
+
```

Comment:
@yispot Please go thru the [template provided for readme.md](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) file and add the required sections.

Reply:
changes are made

Reply:
@yispot You are not following the template, please follow the template and add Publisher and format the doc.